homeassistant/components/bosch_alarm/strings.json

```diff
@@ -22,6 +22,18 @@
           "installer_code": "The installer code from your panel",
           "user_code": "The user code from your panel"
         }
+      },
+      "reauth_confirm": {
+        "data": {
+          "password": "[%key:common::config_flow::data::password%]",
+          "installer_code": "Installer code",
+          "user_code": "User code"
+        },
+        "data_description": {
+          "password": "The Mode 2 automation code from your panel",
+          "installer_code": "The installer code from your panel",
+          "user_code": "The user code from your panel"
+        }
```

Comment:
We can use references to the already present strings

Reply:
whats the proper syntax for that?

Reply:
`"[%key:component::bosch_alarm::config::step::user::data_description::password%]"`